AAP-46705 notification feed edits for AoC (#3858)
ansible/aap-docs: stories/topics/con-aap-notification-feed.adoc

:_mod-docs-content-type: CONCEPT

[id="con-aap-notifications-feed_{context}"]

= Product Notification Feed 

[role="_abstract"]

> Effective July 2025, the {PlatformNameShort} RSS notification feed will be available. 

Reviewer: ```suggestion
Effective July 2025, the {PlatformNameShort} RSS notification feed is available. 
```

This feed serves as a method for communicating various product updates and changes to customers. 

Customers can subscribe to the notifications by visiting link:announcements.ansiblecloud.redhat.com/feed.atom[announcements.ansiblecloud.redhat.com/feed.atom]. 

> Using an RSS feed reader, customers will be updated with events such as {PlatformNameShort} upgrades and system maintenance.

Reviewer: ```suggestion
Using an RSS feed reader, customers are updated with events such as {PlatformNameShort} upgrades and system maintenance.
```

> All {PlatformNameShort} customers can subscribe to this content. Messages include categorization tags to specify deployment types: managed, self-managed (on-prem), or a combination. Red Hat is developing a future enhancement to integrate this feature directly into the UI.

Reviewer: ```suggestion
All {PlatformNameShort} customers can subscribe to this content. Messages include categorization tags to specify deployment types: managed, self-managed (on-premise), or a combination. Red Hat is developing a future enhancement to integrate this feature directly into the UI.
```
